src/index.ts

```diff
@@ -81,6 +87,22 @@ async function getSchemaPaths(
   throw new Error('unreachable'); // `dest` always emits one doc.
 }
 
+// Convenience shortcut for getting the simplified schema.
+async function getSimplifiedSchema(
+  source: Document[] | MongoDBCursor | Readable
+): Promise<SimplifiedSchema> {
+  const streamSource = getStreamSource(source);
+
+  const dest = new PassThrough({ objectMode: true });
+  await pipeline(streamSource, stream({
+    simplifiedSchema: true
+  }), dest);
+  for await (const result of dest) {
+    return result;
+  }
+  throw new Error('unreachable'); // `dest` always emits one doc.
```

Comment:
```suggestion
  throw new Error('unreachable'); // `dest` always emits exactly one doc.
```

(… right?)

Reply:
Yup! Updated, also reduced some of code duplication we had here.